CONTRIBUTING.md

```diff
@@ -14,9 +14,24 @@ Cisco Network Elements support a rich set of features to make networks robust, e
   * Follow all guidelines documented in [README-develop-node_utils-APIs](docs/README-develop-node-utils-APIs.md)
   * Make changes in your branch.
 * Testing
-  * Create a minitest for new APIs or new functionality
-  * Make sure your new minitest file(s) are included in `tests/test_all_cisco.rb`
-  * Run all the tests to ensure there was no collateral damage to existing code
+  * Create a minitest script for any new APIs or new functionality
+  * Run all the tests to ensure there was no collateral damage to existing code. There are two ways you can specify the Nexus switch (virtual or physical) to test against when running the full test suite:
+    1. Use the NODE environment variable to specify the address, username, and password:
+
+        ```bash
+        export NODE="192.168.100.1 user password"
+        rake test
```

Comment:
'rake test' is the way to run the whole test suite. If you want to run a single test file, that's not currently available through rake - so for running an individual script, the old 'ruby' method is still needed.